lib/seedbank/dsl.rb

```diff
@@ -3,7 +3,16 @@ module DSL
     refine Object do
       def override_seed_task(*args)
         task_name, _, deps = Rake.application.resolve_args(args)
-        seed_task = Rake::Task.task_defined?(task_name) ? Rake::Task[task_name].clear : Rake::Task.define_task(task_name)
+
+        seed_task =
+          if Rake::Task.task_defined?(task_name)
+            task = Rake::Task[task_name]
+            deps.unshift('environment') if task.prerequisites.include?('environment')
```

Comment:
It can be `deps.unshift *(task.prerequisites & ['environment'])`
Not sure is it clear to understand but it definitely would not violate line length ☺️